scripts/unicode-identifier.js

```diff
@@ -33,6 +33,10 @@ function tablify(cps) {
 }
 
 console.log(`/** Unicode ${process.versions.unicode} ID_Start/Other_ID_Start ranges */`);
-console.log(`const unicodeIdentifierStart: i32[] = [${tablify(starts)}];\n`);
+console.log(`const unicodeIdentifierStart: i32[] = [${tablify(starts)}];`);
+console.log(`const unicodeIdentifierStartMin = ${starts[0]};`);
+console.log(`const unicodeIdentifierStartMax = ${starts[starts.length - 1]};\n`);
```

Comment:
```suggestion
console.log(`const UnicodeIdentifierStartMin = ${starts[0]};`);
console.log(`const UnicodeIdentifierStartMax = ${starts[starts.length - 1]};\n`);
```

Reply:
Using title case for `SomeEnum.SomeValue` seems fine, but just `SomeValue` conflicts visually with classes, enums and interfaces. Not sure it's preferable?